app/views/maintenance_tasks/runs/_run.html.erb

```diff
@@ -10,6 +10,10 @@
     <%= render "maintenance_tasks/runs/info/#{run.status}", run: run %>
   </div>
 
+  <div class="content">
```

Comment:
+1 Yeah ideally we should change the individual status partials to render the `<div>`s instead too.

Reply:
I don't have a strong opinion on this, it just felt right to me to wrap this into a `div` so it could be easy to differentiate what comes from the engine and what comes from the hosting app. Should we put an `id` for it as well?